elements/lisk-p2p/src/constants.ts

```diff
@@ -66,6 +68,9 @@ export const DEFAULT_MAX_RECONNECT_TRIES = 3;
 
 // Disconnect statuses
 export const INTENTIONAL_DISCONNECT_CODE = 1000;
+export const INVALID_MESSAGE_RECEIVED_CODE = 4009;
```

Comment:
Why 4009 instead of 4109 or 4409?

Reply:
Its a socketcluster specific close code on invalid raw messages,
https://github.com/SocketCluster/socketcluster-server/blob/master/scserversocket.js#L106